### PR TITLE
Feature/merge conflict demo

### PR DESCRIPTION
**Pull request by kushal0007**

### CODE REVIEW

File: newFile.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ this is new file
 i added another file
 another msg
 new line by Santosh
+naya line
```
Comment (bastolaxantos) [inline newFile.txt:5]: What is the purpose of this changes?

Comment (kushal0007) [reply newFile.txt:5]: test
